pinot-controller/src/main/resources/app/pages/TaskDetail.tsx

```diff
@@ -104,9 +104,9 @@ const TaskDetail = (props) => {
           <Grid item xs={12}>
             <strong>Start Time:</strong> {get(taskDebugData, 'startTime', '')}
           </Grid>
-          <Grid item xs={12}>
+          {/* <Grid item xs={12}>
```

Comment:
Please remove the code if it's not going to be used rather than commenting it out.